.github/workflows/codecov.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           persist-credentials: false
       - uses: actions/setup-node@v3
         with:
-          node-version: 18.16.1
+          node-version: 18.18.0
```

Comment:
maybe it makes sense to try 18.15 version if the bug is still not fixed in the che-code?

Reply:
maybe but we're using 18.16.1 downstream in 3.9 ... https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/dependencies/job-config.json#L655-L657 ... https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/DS_CI/job/dsc_3.9/ and that works OK